packages/syftcli/setup.py

```diff
@@ -5,6 +5,7 @@
 __version__ = "0.1.3"
 
 packages = [
+    "docker==6.1.3",
```

Comment:
MAYBE: We could try minimizing adding external packages to the CLI tool, mainly, in the interest of keeping the CLI tool lightweight

But we could add external packages, if they offer a significant amount of functionality for the end user.

I noticed, this package, was not currently being used.

Reply:
Ah yes slipped pass through. Removed the Docker SDK. Though we should be using SDK instead of docker process, but for now removing it anyway.